packages/client/src/app/pages/search/find-blogs/find-blogs.component.html

```diff
@@ -24,7 +24,8 @@ <h3 style="margin-bottom: 0;">No blogs found</h3>
             <div *ngFor="let blog of results.docs | paginate: { id: 'blogs', itemsPerPage: results.limit, currentPage: pageNum, totalItems: results.totalDocs }">
                 <div class="blog-container">
                     <div class="blog-header">
-                        <h3>{{ blog.title }}</h3>
+                        <h3><a [routerLink]="['/portfolio', $any(blog.author)._id, $any(blog.author).username | slugify, 'blog', blog._id]">{{ blog.title }}</a></h3>
```

Comment:
In my testing, I found that blog.author.username returns a result that has capitals and spacing, which is different from how it's used in the URL for the user profile and such.

Reply:
the slugify pipe fixes that by stripping capitals, special characters, and replacing spaces with hyphens